buildSrc/src/main/kotlin/kotlinx.fuzz.example-module.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencies {
 }
 
 kotlin {
-    jvmToolchain(17)
+    jvmToolchain(11)
```

Comment:
Whi is it 11?

Reply:
Some of our example projects are not compatible with jdk 8, as they use some JDK 9+ features. Therefore I just downgraded them to the lowest compatible version of the JDK